documentation/3.4/content/release-notes.md

```diff
@@ -57,6 +58,15 @@ draft: false
 
 ### Change log
 
+#### Operator 3.4.6
+
+* Resolved several issues related to WebLogic Deploy Tooling (WDT) models for Model in Image.
+  * Added Support or provided clearer validation errors for cases where `AdminServerName` is set or missing and where there is or is not a matching entry under `Server`.
+  * Improved support for dynamic clusters that do not specify `ServerTemplate` or `DynamicClusterSize`.
+  * Provided clearer validation errors for configured clusters that have no member servers defined under `DynamicServers`.
```

Comment:
Minor - The clear error message is for dynamic cluster i.e. "Provided clearer validation errors for dynamic clusters".